app/migrations.go

```diff
@@ -17,7 +17,7 @@ const GuestRolesCreationMigrationKey = "GuestRolesCreationMigrationComplete"
 const SystemConsoleRolesCreationMigrationKey = "SystemConsoleRolesCreationMigrationComplete"
 const ContentExtractionConfigDefaultTrueMigrationKey = "ContentExtractionConfigDefaultTrueMigrationComplete"
 const PlaybookRolesCreationMigrationKey = "PlaybookRolesCreationMigrationComplete"
-const FirstAdminSetupCompleteKey = "FirstAdminSetupComplete"
+const FirstAdminSetupCompleteKey = model.SystemFirstAdminSetupComplete
```

Comment:
Ah nice catch, all the above would probably benefit from having them defined as constants too